### ISSUE TITLE
The training process of CNN implemented by Pytorch may have something wrong on the master branch.

### ISSUE THREAD

**Issue by nnnnnzy** [d2l-ai/d2l-en#1917]
In [7.6.2](http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_convolutional-neural-networks/lenet.html#training), [8.1.3](http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_convolutional-modern/alexnet.html#training), [8.2.3](http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_convolutional-modern/vgg.html#training), [8.3.3](http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_convolutional-modern/nin.html#training), and [8.4.3](http://preview.d2l.ai.s3-website-us-west-2.amazonaws.com/d2l-en/master/chapter_convolutional-modern/googlenet.html#training), the loss of CNN implemented by Pytorch has not decreased at all or has not decreased in the first few epochs, which is different from the results of Mxnet and Tensorflow.
